ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -604,3 +606,34 @@ def test_ipa_certs_check_ipacertnsstrust(self):
                 check["kw"]["key"] in DEFAULT_PKI_KRA_CERTS
             )
         tasks.uninstall_master(self.master)
+
+
+class TestIpaHealthCheckOnReplica(IntegrationTest):
```

Comment:
I don't think this needs a separate class (and accompanying server installs). I think this can be added to TestIpaHealthCheck before test_ipa_healthcheck_remove.

Otherwise looks good.